packages/opentelemetry-core/src/metrics/NoopMeter.ts

```diff
@@ -68,33 +68,33 @@ export class NoopMeter implements Meter {
 }
 
 export class NoopMetric<T> implements Metric<T> {
-  private readonly _handle: T;
+  private readonly _instrument: T;
 
-  constructor(handle: T) {
-    this._handle = handle;
+  constructor(instrument: T) {
+    this._instrument = instrument;
   }
   /**
-   * Returns a Handle associated with specified LabelSet.
-   * It is recommended to keep a reference to the Handle instead of always
+   * Returns an Instrument associated with specified LabelSet.
```

Comment:
"Returns a Bound Instrument"

"keep a reference to the Bound Instrument instead"